gcloud-java-core/src/main/java/com/google/cloud/ServiceOptions.java

```diff
@@ -439,6 +441,20 @@ public String applicationName() {
     return APPLICATION_NAME;
   }
 
+  /**
+   * Returns {@code gcloud-java}, the library's name, as a string.
```

Comment:
Fine as is though I like this one slightly better:
`Returns the library's name, {@code gcloud-java}, as a string.`